src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -532,6 +532,19 @@ internal IEnumerable<Autodesk.Revit.DB.GeometryObject> InternalGeometry(bool use
                 }
             }
 
+            // FamilySymbol is a special case: the symbol must be activated before geometry is available, unless an instance is already present in the model
+            else if ((thisElement is FamilySymbol) && (geomElement == null || !geomElement.Any()))
+            {
+                var fs = (FamilySymbol)thisElement;
+                if (!fs.IsActive)
+                {
+                    fs.Activate();
+                    DocumentManager.Regenerate();
```

Comment:
Will this kick off document updates? If so that might end up re-executing the graph as some nodes (and in some cases DynamoRevitModel) are subscribed to document updated events.

Reply:
Yes, I think it will. But the second time it won't call `DocumentManager.Regenerate()` again for the same family symbol, as fs.IsActive is true.